cylc/uiserver/jupyterhub_config.py

```diff
@@ -49,21 +55,35 @@ def _load(path):
         exec(path.read_text())
 
 
+def get_conf_dir_hierarchy(config_paths: List[Path]):
```

Comment:
This file is only loaded by JupyterHub, to open it up to Jupyter Server too need to add it to the list of `config_file_paths` in `cylc.uiserver.CylcUIServer`

Reply:
Need to update? https://github.com/cylc/cylc-uiserver/blob/1b2a5993d41e03e6291e917c91d4d09ee7feeb9c/cylc/uiserver/app.py#L103-L115